src/node/hooks/express/tests.js

```diff
@@ -59,7 +59,11 @@ exports.expressCreateServer = function (hook_name, args, cb) {
   });
 
   args.app.get('/tests/frontend', function (req, res) {
-    res.redirect('/tests/frontend/index.html');
+    res.redirect('/tests/frontend/index.html?target=pad');
```

Comment:
I feel like passing a param here would break if you already pass a param IE ``?grep=bold``

Reply:
It works well even on these routes: `/tests/frontend/*`, `/tests/frontend/specs/*`

Reply:
I mean, have you checked nothing is trying to pass a parameter via the URI?  Because that's something you would need to check and have test coverage for.